falltergeist/src/Game/Game.cpp

```diff
@@ -32,7 +32,7 @@
 #include "../Event/Dispatcher.h"
 #include "../Event/State.h"
 #include "../Exception.h"
-#include "../Format/Gam/File.h"
+#include <falltergeist/io//Gam/File.h>
```

Comment:
does double slash put in these includes for a reason?

Reply:
That's because I had to manually sed some replacements in. It's a bug :(